lib/ably/models/message_encoders/base.rb

```diff
@@ -93,37 +98,38 @@ def strip_current_encoding_part(message)
         #
         # @return [Boolean]
         def is_empty?(message)
-          message[:data].nil? || message[:data] == ''
+          message[:data].nil? || message[:data] == ""
         end
       end
 
       # @api private
       def self.register_default_encoders(client, options = {})
-        binary_protocol = !!options[:binary_protocol]
```

Comment:
Up to you, but I think this rule is useless, only complicates code with extra condition

Reply:
I was trying to fix as many offenses as possible. IMO, both `!!param` and `param ? true : false` are OK.